packages/EasyApiToken/composer.json

```diff
@@ -16,7 +16,7 @@
         "auth0/auth0-php": "^5.4",
         "psr/container": "^1.0",
         "laravel/lumen-framework": "^5.5",
-        "symfony/dependency-injection": "5.1.10",
+        "symfony/dependency-injection": "^5.1.10",
```

Comment:
It was `^4.4 || ^5.1.5` [before the 3.0.5 version](https://github.com/eonx-com/easy-monorepo/blob/3.0.4/packages/EasyApiToken/composer.json). Let's revert it to that restriction back.

Reply:
done